Source/WebInspectorUI/UserInterface/Views/FindBanner.js

```diff
@@ -227,14 +229,17 @@ WI.FindBanner = class FindBanner extends WI.NavigationItem
 
     // Private
 
-    _inputFieldKeyDown(event)
-    {
-        if (event.keyIdentifier === "Shift")
-            this._searchBackwards = true;
-        else if (event.keyIdentifier === "Enter")
-            this._searchKeyPressed = true;
-    }
+    _inputFieldKeyDown(event) {
+        if (event.keyIdentifier === "Enter") {
+            if (this._numberOfResults > 0) {
```

Comment:
please remove this similar code from `_inputFieldInput` as it's not really reachable/needed there anymore

Reply:
Removed in pr.